packages/shylu/shylu_dd/frosch/src/Tools/FROSch_SubdomainSolver_def.hpp

```diff
@@ -346,13 +346,13 @@ namespace FROSch {
     {
         FROSCH_TIMER_START_SUBDOMAINSOLVER(initializeTime,"SubdomainSolver::initialize");
 #ifdef HAVE_SHYLU_DDFROSCH_AMESOS
-        if (!ParameterList_->get("SolverType","Amesos").compare("Amesos")) {
+        if (!ParameterList_->get("SolverType","Amesos2").compare("Amesos")) {
```

Comment:
All these changes seem to be unnecessary. They could be circumvented but not relying on default return values (as indicated in one of the other review comments on this PR).

Reply:
See above.